### DISCUSSION TITLE
Does this support Windows?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [blueman-project/blueman#2023] [Q&A] [answered]
Having a cross-platform Bluetooth device manager would be brilliant, but the choice of GTK instead of Qt causes me to worry (indeed, despite GIMP and Fedora Media Writer achieving cross-platform support).

**Comment by cschramm** [comment, marked as answer]
blueman is basically a frontend to BlueZ, so it is tightly coupled to Linux.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/blueman-project/blueman/discussions/2023#discussioncomment-4874291

@cschramm, do you know of a cross-platform alternative?

**Reply by cschramm** [reply]
I don't and I even doubt that there is any third-party Bluetooth GUI on Windows as it does not sound very feasible and the fields is taken by the system default. I have no idea about Bluetooth interfaces (both user-facing and API- / kernel-level) on BSD systems.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/blueman-project/blueman/discussions/2023#discussioncomment-4874291

Relevantly, I have found https://github.com/bluez/bluez/issues/591#issuecomment-1701918798, which answers this question at that level, too.
